pkg/api/chunk.go

```diff
@@ -169,6 +169,21 @@ func (s *Service) chunkUploadHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	err = putter.Put(r.Context(), chunk)
+	if err != nil {
+		logger.Debug("chunk upload: write chunk failed", "chunk_address", chunk.Address(), "error", err)
+		logger.Error(nil, "chunk upload: write chunk failed")
+		switch {
+		case errors.Is(err, postage.ErrBucketFull):
+			jsonhttp.PaymentRequired(ow, "batch is overissued")
```

Comment:
maybe more clear error msg is appropirate: `the postage batch is overissued and cannot accept more chunks`?

Reply:
I think "batch is overissued" is enough - concise.